### PR TITLE
hn _locales : message.json [HINDI]

### LINKED ISSUE #3550: Translate to Hindi
Guide to translating:
https://github.com/MetaMask/metamask-extension/blob/master/docs/translating-guide.md

### PR DESCRIPTION
**Pull request by itsN1X**
edit: 

#### Updated till [here](https://github.com/MetaMask/metamask-extension/pull/3611/commits/6a41c165fd317fe649b9561efb488791d2064e3d). 

cc/ @kumavis @danfinlay 

---

Please email me @vs77bb , I'm new to gitcoin.co, ~~`newbie`~~`coder` as it currently labels me [here](https://gitcoin.co/profile/itsn1x/). It'd be really great to be guided over the whole smart contract bounty extraction process.

Fixes #3550 . 

Cheers <3
~ N1X
<`0xD0f79B71A8ffB7f70392630f8BFc900fcA27af42`>$ETH.MetaMask

### CODE REVIEW

File: app/_locales/hn/messages.json [hunk 1]
```diff
@@ -0,0 +1,819 @@
...
+    "message": "आपका डेन मेटामास्क के भीतर आपका पासवर्ड-एन्क्रिप्टेड स्टोरेज है।"
+  },
+  "deposit": {
+    "message": "जमा"
+  },
+  "depositBTC": {
+    "message": "नीचे दिए गए पते पर अपना बीटीसी जमा करें:"
+  },
+  "depositCoin": {
+    "message": "नीचे दिए गए पते पर अपना $ 1 जमा करें",
```
Comment (kumavis) [inline app/_locales/hn/messages.json]: `$1` is a variable and must be written exactly this way

File: app/_locales/hn/messages.json [hunk 2]
```diff
@@ -0,0 +1,819 @@
...
+  },
+  "getEther": {
+    "message": "ईथर प्राप्त करें"
+  },
+  "getEtherFromFaucet": {
+    "message": "$1 के लिए एक नल से ईथर प्राप्त करें",
+    "description": "ईथर नल के लिए नेटवर्क नाम प्रदर्शित करता है"
+  },
+  "greaterThanMin": {
+    "message": "$ 1 के बराबर या बराबर होना चाहिए।",
```
Comment (kumavis) [inline app/_locales/hn/messages.json]: `$1` is a variable and must be written exactly this way

File: app/_locales/hn/messages.json [hunk 3]
```diff
@@ -0,0 +1,819 @@
...
+    "description": "एक खाता आयात करने के लिए प्रारूप"
+  },
+  "kovan": {
+    "message": "कोवान टेस्ट नेटवर्क"
+  },
+  "knowledgeDataBase": {
+    "message": "हमारे नॉलेज बेस पर जाएं"
+  },
+  "lessThanMax": {
+    "message": "$ 1 से कम या बराबर होना चाहिए।",
```
Comment (kumavis) [inline app/_locales/hn/messages.json]: `$1` is a variable and must be written exactly this way

File: app/_locales/hn/messages.json [hunk 4]
```diff
@@ -0,0 +1,819 @@
...
+    "message": "उपयोग की शर्तें"
+  },
+  "testFaucet": {
+    "message": "टेस्ट नलि"
+  },
+  "to": {
+    "message": "के लिए"
+  },
+  "toETHviaShapeShift": {
+    "message": "शैपशैफ्ट द्वारा $ 1 से ETH में जमा",
```
Comment (kumavis) [inline app/_locales/hn/messages.json]: `$1` is a variable and must be written exactly this way

**Review by kumavis — CHANGES_REQUESTED** [review]

**Comment by kumavis** [comment]
@itsN1X looks great, just need a couple changes
we'll make sure you are able to redeem the bounty

File: app/_locales/hn/messages.json (hunk 1 above)
Comment (itsN1X) [reply app/_locales/hn/messages.json]: Oh! Sorry that escaped my conscience the other night!
`variable` correction enroute ( :

File: app/_locales/hn/messages.json (hunk 4 above)
Comment (itsN1X) [reply app/_locales/hn/messages.json]: @kumavis - [x] Made the `$1` behave as `variable`.
---
Any further changes required? Review me!

**Comment by itsN1X** [comment]
@kumavis , have a look !

**Review by kumavis — APPROVED** [review]

**Comment by itsN1X** [comment]
<3

**Pull request merged**